CMakeLists.txt

```diff
@@ -20,6 +20,8 @@ cmake_minimum_required(VERSION 3.14)
 
 project(Irrlicht LANGUAGES CXX C)
 
+add_definitions("/D_ENABLE_EXTENDED_ALIGNED_STORAGE")
```

Comment:
will take care of it when I'm done with cache

Reply:
ok, thanks for pointing out!